src/ui/blockUi-editor-audioBlockEditor.js

```diff
@@ -16,14 +16,24 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
     // an editing interface for individual audio-type blocks
     fluid.defaults("sjrk.storyTelling.blockUi.editor.audioBlockEditor", {
         gradeNames: ["sjrk.storyTelling.blockUi.editor.mediaBlockEditor"],
+        selectors: {
+            previewPlaceholder: ".sjrkc-st-block-audio-preview-placeholder"
+        },
+        modelListeners: {
```

Comment:
I tried to determine why this was only in place for the audio block editor, and pored over my notes and change logs from when I added this model listener (back on May 20), and it seems that at the time the audio block was the only type that required its own explicit model listener to hide and show the preview placeholder image (the little microphone icon) along with the preview element itself (in this case, the `audio` element).

Since then, and confirmed by the testing I've just done, video block preview placeholders aren't hiding with their `video` elements. Tangentially related to this, Chrome has started to exhibit the symptoms of [SJRK-226](https://issues.fluidproject.org/browse/SJRK-226), which previously only applied to Firefox. With this in mind, I have added this model listener to **all** media block types (`audio`, `image`, and `video`, currently), and have also solved SJRK-226 by making the image block preview placeholder its own element (the same one shared by the other two types).

Furthermore, because the `imageBlockEditor` grade is currently not derived from the same base as the `audioBlockEditor` and `videoBlockEditor` grades (which are both instances of `mediaBlockEditor`), this solution required a bit of duplication. The work to combine these three block type editors into `mediaBlockEditor` is described in [SJRK-175](https://issues.fluidproject.org/browse/SJRK-175), which will hopefully get some attention in the future, but may be out of scope for this issue.

Please see the commit message for this commit for more details: https://github.com/fluid-project/sjrk-story-telling/pull/77/commits/c539dd0f0b3b9554b6e72affdd770d87ab3301b9